game/assets/jak1/subtitle/game_subtitle_it.gd

```diff
@@ -0,0 +1,2605 @@
+  (512)
+  (523 "???" "MI SERVIREBBERO SOLO 20 BATTERIE PER AVERE ABBASTANZA ENERGIA")
+  (612 "???" "DA RESISTERE AL CALORE DEL CANYON! VERO PAPÀ?")
+  (696)
+  (707 "SAMOS" "CERTO KEIRA, POTREBBE FUNZIONARE...")
+  (786 "SAMOS" "MA, QUESTO GIOVANE RAGAZZO,")
+  (838 :offscreen "SAMOS" "E LA CREATURA!")
+  (875 "SAMOS" "DOVE TROVERANNO 20 BATTERIE?")
+  (923 "KEIRA" "LÀ, AL VILLAGGIO!")
+  (961 :offscreen "KEIRA" "MOLTI LAGGIÙ HANNO UNA O DUE BATTERIE NASCOSTE DA QUALCHE PARTE.")
+  (1059 :offscreen "KEIRA" "ANCHE SE LORO NON SARANNO DELL'IDEA DI SCAMBIARLE,")
+  (1129 "KEIRA" "METTETE NELLE LORO MANI QUALCHE GLOBO PRECURSOR E RIUSCIRETE A RAGGIRARLI!")
+  (1243)
+  (1257 :offscreen "KEIRA" "E SCOMMETTO CHE CE NE SONO MOLTE DI PIÙ ABBANDONATE NEL DESERTO IN ATTESA CHE QUALCHE")
+  (1366 "KEIRA" "PRODE AVVENTURIERO VADA A RECUPERARLE!")
+  (1447 "DAXTER" "BEH, ECCO FINALMENTE IL CORAGGIOSO AVVENTURIERO!")
+  (1529)
+  (1538 "SAMOS" "\"CORAGGIOSO AVVENTURIERO\"?")
+  (1589 "SAMOS" "VOI DUE NON RIUSCIRESTE A USCIRE DAL VILLAGGIO, SENZA ALLENAMENTO!")
+  (1685)
+  (1695 "SAMOS" "PRIMA DI FARE QUALUNQUE ALTRA COSA,")
+  (1757 "SAMOS" "FARESTE MEGLIO AD ANDARE AL TELETRASPORTO PER ALLENARVI SULLA ROCCIA DEL GEYSER!")
+  (1872)
+  (1882 "DAXTER" "EHM, NON È CHE TROVEREMO ANCORA DELL'ALTRA BRUTTA ROBACCIA OSCURA, VERO?")
+  (1996 "DAXTER" "NON VORREI CADERCI DENTRO DI NUOVO E DIVENTARE COME TE!")
+  (2069)
+  (2072 "SAMOS" "ENTRATE SUBITO LÌ, PRIMA CHE VI TRASFORMI ENTRAMBI IN FELCI!")
+  (2189)
+  )
+
+("sage-intro-sequence-e"
+  (205 "SAMOS" "OTTIMO ALLENAMENTO! MA QUESTO NON È NULLA IN CONFRONTO ALLE SFIDE CHE ANCORA VI ATTENDONO!")
+  (361 "DAXTER" "NAH, NON C'È NESSUUUUN PROBLEMA!")
+  (438 "DAXTER" "DOBBIAMO DARCI UNA MOSSA, VERO JAK?")
+  (496 "DAXTER" "CI PIACEREBBE RESTARE A PARLARE CON L'OMINO VERDE, MA ORA")
+  (589 "DAXTER" "DOBBIAMO RIPRENDERE LE NOSTRE FATICHE!")
+  (660 "SAMOS" "BENE, BENE. ALLORA PARTITE PER \"L'AVVENTURA\"...")
+  (787 "SAMOS" "E MENTRE VIAGGIATE, IN CERCA D' \"AVVENTURA\", CERCATE ALMENO DI RENDERVI UTILI!")
+  (940)
+  (957 :offscreen "SAMOS" "QUEI RACCOGLITORI DI ECO VERDE SI SONO INTASATI DI NUOVO.")
+  (1043)
+  (1058 :offscreen "SAMOS" "ANDATE ALL'ESTREMITÀ PIÙ LONTANA DELLA SPIAGGIA E DATEGLI UNA PULITA.")
+  (1160)
+  (1178 :offscreen "SAMOS" "SEGUITE LE LAMPADE: VI CONDURRANNO FIN LAGGIÙ.")
+  (1310)
+  (1329 :offscreen "SAMOS" "E ORA, TUTTI QUANTI...")
+  (1415)
+  (1439 :offscreen "SAMOS" "FUORI DI QUIIIII!")
+  )
+
+("sidekick-human-intro-sequence-b"
+  (316 "JAK" "SHHH!")
+  (349)
+  (556 "???" "DEVE CONTINUARE LA TUA RICERCA DI ARTEFATTI E ECO.")
+  (659 "???" "GAAAH!")
+  (680 "???" "SE GLI INDIGENI HANNO DEGLI OGGETTI PRECURSOR...")
+  (760 "???" "EH - SAI BENE CHE COSA FARE!")
+  (819)
+  (844 "???" "REGOLARE I CONTI CON CHIUNQUE OSI ALLONTANARSI DAL VILLAGGIO.")
+  (946 "???" "PER IL RESTO LO ATTACCHEREMO A TEMPO DEBITO!")
```

Comment:
Make a distinction between Gol and Maia here, otherwise you can't tell them apart.